### PR TITLE
Add icon to IntelliJ

### PR DESCRIPTION
**Pull request by DRSchlaubi**
@HopeBaron would be great if you could replace the svg with the actual original

### CODE REVIEW

**Comment by lukellmann** [comment]
> @HopeBaron would be great if you could replace the svg with the actual original

where did you get this icon from then?

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 .gradle/
-.idea/
+.idea/*
 out/
 
 **/build/*
 !**/build/generated/
 # re-exclude BuildConfigGenerated.kt
 common/build/generated/sources/buildConfig/main/dev/kord/common/BuildConfigGenerated.kt
+!.idea/icon.svg
```
Comment (lukellmann) [inline .gitignore:10]: i think the include should be put right under the `.idea/*` exclude

**Comment by DRSchlaubi** [comment]
> > @HopeBaron would be great if you could replace the svg with the actual original
> 
> where did you get this icon from then?

Used a png to svg converter

**Comment by lukellmann** [comment]
ok, that explains this:
kord

**Comment by DRSchlaubi** [comment]
> ok, that explains this: kord

yeah exactly

**Comment by lukellmann** [comment]
i now simply replaced the svg with a png, is that also ok @DRSchlaubi?

**Comment by DRSchlaubi** [comment]
If IntelliJ can read it yeah ig

**Comment by lukellmann** [comment]
[kotlinx.coroutines](https://github.com/Kotlin/kotlinx.coroutines/blob/master/.idea/icon.png) and [Kotlin](https://github.com/JetBrains/kotlin/blob/master/.idea/icon.png) also use png, so it seems to work

**Comment by DRSchlaubi** [comment]
alright

**Pull request merged**